hassio/src/addon-view/config/hassio-addon-config.ts

```diff
@@ -94,6 +119,17 @@ class HassioAddonConfig extends LitElement {
             ? ""
             : html` <div class="errors">Invalid YAML</div> `}
         </div>
+        ${hasHiddenOptions
+          ? html`<ha-formfield
+              label="Show unused optional configuration options"
+            >
+              <ha-checkbox
```

Comment:
I think this should be toggle instead of a checkbox

Reply:
I don't agree, but ok.

Reply:
Haha why?

Reply:
The checkbox feels more natural to me, but I don't care enough to challenge you on it, has been switched to switch